contract/src/main/kotlin/io/provenance/scope/loan/LoanScope.kt

```diff
@@ -69,7 +75,7 @@ data class LoanPackage(
     /** Servicing data for the loan, including a list of metadata on loan states. */
     @Record(LoanScopeFacts.servicingData) var servicingData: ServicingData,
     /** A list of third-party validation iterations. */
-    @Record(LoanScopeFacts.loanValidations) var loanValidations: LoanValidation,
+    @Record(LoanScopeFacts.loanValidationMetadata) var loanValidations: LoanValidation,
```

Comment:
I think it'd be nice if we can establish `LoanPackage` as the model of an "up-to-date" loan scope, but I'm open to keeping the old record in this data class as a deprecated record if we think it's important to backwards compatibility

Reply:
I lean toward `LoanPackage` being the current loan scope until usage builds to the point that it would be too hard to not have backwards compatibility.